test/spec/provider/cloud-element-templates/fixtures/complex.json

```diff
@@ -1,5 +1,6 @@
 [
   {
+    "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
```

Comment:
Should we add a testCase which retrieves the schema not from unpkg but from `https://www.jsdelivr.com/` or so?

Reply:
Good catch, I added it accordingly 👍 

https://github.com/bpmn-io/bpmn-js-properties-panel/pull/586/files#diff-a7d860758af72e6e3dc82603905764d2c9ce5c28e93dca29dc9e5efb91fd2bf9